sru/sru_functional.py

```diff
@@ -265,7 +280,8 @@ def __init__(self,
         self.register_buffer('scale_x', torch.FloatTensor([0]))
 
         if layer_norm:
-            self.layer_norm = nn.LayerNorm(input_size)
+            # Use the true input_size here
+            self.layer_norm = nn.LayerNorm(self.input_size)
```

Comment:
`self.layer_norm = nn.LayerNorm(self.input_size if self.custom_u is None else self.output_size)`

Reply:
since you would apply a linear project when input_size != output size and custom_u is provided